app/compile/template/index.js

```diff
@@ -69,12 +69,29 @@ export default async function template(files, options = {}) { // eslint-disable-
   app.option('cwd', options.project_root)
 
   const template_files = files.filter((file) => utils.type(file) === 'template') // find only the template files
+  const project_files = files.filter((file) => file.includes(project_root))
+
   addTemplateLanguages(app, options, template_files) // adds all the template languages available
 
   let initial_resources = await resolveContent(files) // gets the content to use for the project for the initial run
+  let all_locales = [ ...to.keys(initial_resources) ]
+  let locales_from_folders = false
+
+  for (const file of project_files) {
+    const parts = file.split(path.sep)
+    const locale = parts[parts.indexOf('locales') + 1]
```

Comment:
This not going to work correctly if there's no index of `locales` in the file path.

If `locales` doesn't exist in the `parts` then it's going to use `parts[0]` because the `indexOf` would have returned `-1`. You have to have an if statement wrapping most of the code in this for loop checking to make sure that the index is `> -1`